### DISCUSSION TITLE
[ADGUARD HOME] Block Elements | Raspberry PI

### DISCUSSION THREAD

**Discussion by ITRecords38** [AdguardTeam/AdGuardHome#5822] [Q&A]
Hi,

I want to be able to block advertising elements on the sites I visit.

Can we do it and if so I would like some information.

Thanks

**Comment by ghost** [comment]
You're limited to filtering content with syntax's compatible with DNS.

You can read more [here](https://adguard-dns.io/kb/general/dns-filtering-syntax/) about specifics.

For cosmetic filtering, you will need to use either the AdGuard Ad Blocker apps or AdGuard Browser Extensions which are installed on each individual client.

**Comment by ghost** [comment]
https://github.com/AdguardTeam/AdGuardHome/issues/1228

**Comment by ITRecords38** [comment]
Hi @jslawler @akeyakey,

Thanks for your feedback. I will look at the proxy part and that corresponds to what I want :)
